core/src/components/popover/popover.tsx

```diff
@@ -208,6 +208,15 @@ export class Popover implements ComponentInterface, PopoverInterface {
    */
   @Prop() isOpen = false;
 
+  /**
+   * If `true`, the popover will cancel browser keyboard event bindings to prevent
+   * scroll behavior in a popover using a list of items.
+   *
+   * Disable this behavior to allow the contents of the popover to handle their own
+   * keyboard interactions (i.e. when using CSS scroll snap).
+   */
+  @Prop() cancelBrowserKeyboardEvents = true;
```

Comment:
What do you think about making this a prop that disables the keyboard events when `true`? I think if developers want the browser keyboard events to be handled normally, it's likely they don't want Ionic getting in the way either. I can't think of a use case where we would want both the browser and Ionic to handle the keyboard events.

Reply:
Makes sense to me! Verified locally that all the keyboard navigation built-ins work the same after disabling the popover's keyboard event handlers.

I went with `disableKeyboardEvents` for the property, open to suggestions. We don't appear to have a definite pattern for enable vs. disable language for properties.

Reply:
I try to name public APIs using positive language, that way it's easier to reason about when negating values. For example `!popoverEl.enableKeyboardEvent` is easier to reason about than `!popoverEl.disableKeyboardEvent`